maps/randomvaults/objects.dm

```diff
@@ -136,3 +139,20 @@
 /obj/item/weapon/paper/iou
 	name = "paper- 'IOU'"
 	info = "I owe you a rod of destruction. Redeemable at Milliway's at the end of time."
+
+/obj/machinery/floodlight/on
+
+	New()
```

Comment:
No it's just shit code to do relative pathing and I can't believe it's not clusterfack catching this